src/Sql/Sql/help/Update-AzSqlInstanceDatabaseVulnerabilityAssessmentSettings.md

```diff
@@ -223,7 +223,7 @@ Parameter Sets: UpdateByInputObjectParameterSet
 Aliases:
 
 Required: True
-Position: Named
+Position: 0
```

Comment:
I not sure why you touched this file but it's a breaking change.

Reply:
Fix